README.md

```diff
@@ -10,3 +10,5 @@ to ROS standard msgs and publishes it again.
 There are multiple Radar sources, so a node should be run per input/output pair.
 
 And a launch file to bring them all, and in the darkness bind them.
+
+This packages is currently tailored to work with the following ipab-rad packages: [radar](https://github.com/ipab-rad/radar/tree/init_config) and [ros1_bridge](https://github.com/ipab-rad/radar/tree/init_config)
```

Comment:
The ros1_bridge link should be https://github.com/ipab-rad/ros1_bridge.

Also best not link to branches for posterity, since we're aiming to merge those branches and remove them soonTM.

Reply:
True, will change for the std URLS

Reply:
The correct URLs are used now. Also a "Usage" section has been added to the README